Assets/MRTK/Providers/Oculus/XRSDK/MRTK.ThirdParty/MRTK-Quest.meta

```diff
@@ -0,0 +1,8 @@
+fileFormatVersion: 2
```

Comment:
Feels like this should be a peer of XRSDK (likely renamed to XR2018, to maintain parity with other folders)

Reply:
That presumes we are planning on supporting MRTK-Quest for the Unity 2018 XR API